Tribler/community/tunnel/tunnel_community.py

```diff
@@ -926,7 +925,7 @@ def on_create(self, messages):
             self.tunnel_logger.info(
                 'TunnelCommunity: we joined circuit %d with neighbour %s', circuit_id, candidate.sock_addr)
 
-            shared_secret, Y, AUTH = self.crypto.generate_diffie_shared_secret(message.payload.key)
+            shared_secret, y, auth = self.crypto.generate_diffie_shared_secret(message.payload.key)
```

Comment:
Ah ok, pycharm was giving a lint warning about it. But as @whirm mentioned we'll save this for later :smile: